frontend/src/components/Navbar.vue

```diff
@@ -0,0 +1,28 @@
+<template>
+    <Nav>
+        <Header />
+        <Sidepanel />
+    </Nav>
+</template>
+
+<script>
+import Header from './Header.vue';
+import Sidepanel from './Sidepanel.vue';
+    export default {
+        components: { Header, Sidepanel },
+        data: () => ({
+            drawer: true
+        })
+    }
+</script>
+
+<style lang="scss">
+    .v-navigation-drawer__border {
```

Comment:
whenever i try scoping this rule or moving it to the corresponding component, it stops affecting the target element, do you have any idea what could be the cause?

Reply:
probably because you are trying to reach class name, that defined in child components https://vue-loader.vuejs.org/guide/scoped-css.html#child-component-root-elements you need to use `deep` or `::v-deep` selector in such cases